### PR TITLE
Feature/add service

### PR DESCRIPTION
**Pull request by decaillet**

### CODE REVIEW

File: unit_tests/test_service.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import unittest.mock
+
+from src import *
+
+
+class TestService(unittest.TestCase):
+    def test_run(self) -> None:
```
Comment (BenjaminDecaillet) [inline unit_tests/test_service.py:7]: maybe call it "test_service" if we want to have multiple test cases for Service at the end

File: README.md [hunk 2]
```diff
@@ -54,5 +56,11 @@ mkvirtualenv m05-mp-decaillet --python "C:\Users\%username%\AppData\Local\Progra
 ```
 
 _NB: exact path to **python3.11** may vary; locate it with: `where python` (Windows CMD) or `get-command python` (Windows PowerShell)_
-  
+
+<!-- -------------------------------------------------- -->
+
+## Unit tests
+* activate your virtualenv: `workon subtitles-creator`
+* run unit tests: `python -m unittest discover -v`
```
Comment (BenjaminDecaillet) [inline README.md:64]: We could add the "-s 'unit_tests" for the command to run for the whole directory

**Review by BenjaminDecaillet — APPROVED** [review]

File: unit_tests/test_service.py (hunk 1 above)
Comment (decaillet) [reply unit_tests/test_service.py:7]: in fact "test_runs" as the test name is supposed to say what service does: it runs...

will change and re-submit

File: README.md (hunk 2 above)
Comment (decaillet) [reply README.md:64]: no. it's a filter to run only certain dirs.
you're missing an __init__.py file

**Pull request merged**